mcache/tests/test_check.py

```diff
@@ -134,14 +135,56 @@ def memcached():
 
     yield
 
-    subprocess.check_call(["docker-compose", "-f", os.path.join(HERE, 'docker-compose.yaml'), "down"])
+    subprocess.check_call(["docker-compose", "-f", docker_compose_file, "down"])
+
+
+@pytest.fixture(scope="session")
```

Comment:
Can we separate the pytest fixture and put them in a `conftest.py` to increase readability